### DISCUSSION TITLE
How to properly add/extend CSS?

### DISCUSSION THREAD

**Discussion by QuietSwami** [adityatelange/hugo-PaperMod#1592] [Q&A] [answered]
Hi everyone!

I'm trying to add a custom piece of CSS to create an alert for draft pages.
I've already had the HTML part working, but cant seem to figure out how to insert the required CSS.

I've tried creating a new CCS file inside of the site (not the themes) asset folder (assets/css/style.css), but no luck. What am I missing?

**Comment by jarp0l** [comment, marked as answer]
Maybe try moving the CSS file to `assets/css/extended/` directory as mentioned on [wiki](https://github.com/adityatelange/hugo-PaperMod/wiki/FAQs#bundling-custom-css-with-themes-assets).
